configure.ac

```diff
@@ -475,6 +475,30 @@ AC_ARG_WITH(device,
 	AC_HELP_STRING([--with-device=name], [Specify the communication device for MPICH]),,
 	with_device=default)
 
+if test $with_device = "default" -o $with_device = "ch4" ; then
```

Comment:
I suppose we can just put this inside the AC_ARG_WITH. Let me update again...

Reply:
Actually I think that would be worse for downstream maintainers with different device defaults/options. I'll still with the `if` block for now.